samples/stock/interactive-docs/navigator-styling/demo.html

```diff
@@ -0,0 +1,18 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container" class="chart"></div>
+<div id="button-box">
+    <button id="toggle-mask">Set mask to outside</button><br><br>
+
+    <label for="navigator-mask-fill">Change navigator mask fill:</label>
+    <input type="text" id="navigator-mask-fill"><br><br>
```

Comment:
How about we change the input to a color picker? With a hardcoded opacity (0.3)? I mean, users can change the color, but not the opacity.